### ISSUE TITLE
Problem saving images

### ISSUE THREAD

**Issue by alk1895** [gildas-lormeau/SingleFile#266]
Describe the bug
Problem with images when saving page, before the latest updates, it worked fine.

To Reproduce
Example page link: [https://tmofans.com/](url) (chat box)
Current configuration (Same configuration worked before):
```json
{
  "profiles": {
    "__Default_Settings__": {
      "autoClose": false,
      "autoSaveDelay": 1,
      "autoSaveLoad": false,
      "autoSaveLoadOrUnload": true,
      "autoSaveRepeat": false,
      "autoSaveRepeatDelay": 10,
      "autoSaveUnload": false,
      "backgroundSave": false,
      "browserActionMenuEnabled": true,
      "compressCSS": false,
      "compressHTML": false,
      "confirmFilename": false,
      "confirmInfobarContent": false,
      "contextMenuEnabled": true,
      "credentials": "include",
      "displayInfobar": true,
      "displayStats": false,
      "filenameConflictAction": "uniquify",
      "filenameMaxLength": 192,
      "filenameReplacementCharacter": "_",
      "filenameTemplate": "{page-title}.html",
      "groupDuplicateImages": false,
      "includeInfobar": false,
      "infobarTemplate": "{page-title} (\"{url-href}\" {date-iso})",
      "loadDeferredImages": false,
      "loadDeferredImagesBlockCookies": true,
      "loadDeferredImagesBlockStorage": false,
      "loadDeferredImagesMaxIdleTime": 1500,
      "logsEnabled": true,
      "maxResourceSize": 500,
      "maxResourceSizeEnabled": false,
      "progressBarEnabled": true,
      "referrerPolicy": "origin-when-cross-origin",
      "removeAlternativeFonts": false,
      "removeAlternativeImages": false,
      "removeAlternativeMedias": false,
      "removeAudioSrc": true,
      "removeFrames": false,
      "removeHiddenElements": false,
      "removeImports": false,
      "removeScripts": true,
      "removeUnusedFonts": false,
      "removeUnusedStyles": false,
      "removeVideoSrc": true,
      "saveRawPage": false,
      "saveToClipboard": false,
      "shadowEnabled": true,
      "tabMenuEnabled": true
    }
  },
  "rules": [],
  "maxParallelWorkers": 8
}
```

Screenshots
This image shows the page src:
![image](https://user-images.githubusercontent.com/53287403/64082272-68851600-ccda-11e9-8980-c20db4071a11.png)

The following image shows the src of the saved page:
![image](https://user-images.githubusercontent.com/53287403/64082273-7044ba80-ccda-11e9-9012-25f4ebf60968.png)

Environment
 - OS: Windows 10 x64
 - Browser: Chrome at the last version

**Comment by gildas-lormeau** [comment]
Thank you very much for the detailed report. I was able to reproduce and fix this regression and in the version 1.11.35 I've just published. I'm sorry for the inconvenience.

**Comment by alk1895** [comment]
Thank you for your hard work!
